SEAD Sweep threat buffer distance

MetalStormGhost opened the pull request:
Added a new option in settings: SEAD Sweep threat buffer distance (nmi).

MetalStormGhost committed:
SEAD Sweep threat buffer distance

Added a new option in settings: SEAD Sweep threat buffer distance (nmi).

Raffson commented:
Wouldn't it make more sense to use this setting for the SEAD Sweep's engagement distance? That default threat range of 5NM is simply to cope with packages where the target doesn't have a threat range, in which case it would make sense for SEAD (Sweep) flights to fly in closer.

MetalStormGhost committed:
Added new option in settings: SEAD Sweep engagement range (nmi).

Renamed SEAD Sweep threat buffer distance (nmi) to SEAD Escort/Sweep threat buffer distance (nmi) since it also applies to SEAD Escort flights.

Raffson reviewed (approved):
That works too I guess 😅
